spec/rails_helper.rb

```diff
@@ -73,4 +73,7 @@
   config.filter_rails_from_backtrace!
   # arbitrary gems may also be filtered via:
   # config.filter_gems_from_backtrace("gem name")
+
+  # Include IntegrationHelper module for integration tests via devise
+  config.include Devise::Test::IntegrationHelpers, type: :feature
```

Comment:
Nice one. Funnily enough, I was failing to implement this earlier today myself and knew I was pulling a major brainfreeze.